k8s/notify-stale-prs.yml.j2

```diff
@@ -21,3 +20,6 @@ secretFile:
           minimum_number: 17788
         - name: "Chia-Network/chia-blockchain-gui"
           minimum_number: 2300
+      label_skip_users:
```

Comment:
I think this config key is actually supposed to be skip users. And there's still an old `check_stale_pending_repos` key above this line too.

Reply:
fixed